scripts/index.js

```diff
@@ -0,0 +1,61 @@
+let tempValue = document.getElementById("tempValue");
+let temperature = 63;
+let landscape = document.getElementById("landscape")
+let skySelect = document.getElementById("skySelect")
+let skyElement = document.getElementById("sky")
+
+let increaseTempControl = document.getElementById("increaseTempControl");
+increaseTempControl.addEventListener("click", function () {
+  temperature += 1;
+  tempValue.innerHTML = temperature;
+  if (temperature >= 80) {
+    tempValue.className = "red";
```

Comment:
You should avoid class names like yellow and instead use something like warm or hot.

A class name should describe what the thing means, not its effect.

The example will be updated to reflect this for next cohort.